src/mmw/js/src/draw/views.js

```diff
@@ -77,7 +77,7 @@ function actOnLayer(datum) {
 function validateRwdShape(result) {
     var d = new $.Deferred();
     if (result.watershed) {
-        if (result.watershed.features[0].geometry.type === 'MultiPolygon') {
+        if (result.watershed.geometry.type === 'MultiPolygon') {
```

Comment:
This is still a little strange to me.  We can analyze MultiPolygons in other situations (including directly to the API, for example `wkaoi=huc12__70859`).  Is this check really a shorthand for some other kind of problem with RWD derived shapes?

Reply:
Another example, here is an RWD result which the app won't analyze, but the API will (when the RWD output is given to `land`):

```json
{"location":[39.978299876162005,-75.46028137207031],"snappingOn":true,"dataSource":"drb"}
```
Which comes from the lake-island in your example above.

Reply:
Good point. Going to remove this check. It was added in https://github.com/WikiWatershed/model-my-watershed/pull/1468 when we were trying to guard against the bug `utils.toMultiPolygon` was creating.